sdk/README.md

```diff
@@ -138,53 +153,39 @@ will be listed below.
 - [Exit Handler](/sdk/FEATURES.md#exit-handler)
 
 
-## Additional Features
+## Removable Features
```

Comment:
"Removable" sounds very odd here. "Deprecated" maybe? Maybe "Optional" or keep "Additional" even if it means a feature being removed

Reply:
Sure I can rename it, here I want to say features that can be opt-out (similar to the type check and telemetry flag in the original kfp)

Reply:
Maybe "Deactivating Special Features" or "Capabilities that can be disabled"

Reply:
I rename it to optional features since that sounds a bit better.